recipe/meta.yaml

```diff
@@ -99,8 +99,14 @@ outputs:
         - numba >=0.58.1
         - attrs >=22.2
         # Keep this in sync with TileDB-SOMA's somacore version requirement.
-        - somacore ==1.0.26
+        - somacore ==1.0.28
         - scanpy >=1.9.2
+        # See TileDB-SOMA's apis/python/requirements_spatial.txt
```

Comment:
I thought we were saving the spatial experiment for a later date. spatialdata 0.2.5 still isn't available from conda-forge (I'm actively working on it though)

Reply:
You're right. I made a mistake & copied too much in. I'll fix this.

Reply:
(These PRs are both in draft)